.travis.yml

```diff
@@ -12,19 +12,19 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=latest
         - CS_CHECK=true
         - BENCHMARKS=true
         - TEST_COVERAGE=true
-    - php: 7.4
+    - php: 8.0
       env:
         - DEPS=lowest
-    - php: 7.4
+    - php: 8.0
```

Comment:
We should have this in allowed failures

Reply:
We're doing it on stream at https://twitch.tv/spabby - come and tell us how.